### DISCUSSION TITLE
better wire placement

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#7463] [Ideas and suggestions]
placing a component on an existing wire should split the wire.

example:

![image](https://user-images.githubusercontent.com/87497640/143090235-5c7f19fc-4887-4892-a907-106ae9951784.png)
a complex wire pattern that would be a pain to recreate ^

![image](https://user-images.githubusercontent.com/87497640/143090437-c990be10-35dc-4ea9-b51b-eaae08477da1.png)
the wire is split in 2, preserving the complex shape and routing, also automatically connecting the wires to available component slots 

![image](https://user-images.githubusercontent.com/87497640/143090627-dbb43937-fc1d-40d2-b347-61e0e450f16a.png)
the wire splitting in half, leaving 2 ends to be manually attached ^

either way would be good, but automatic would be even gooder.

a wire would be consumed from inventory upon splitting.

**Comment by MaaiG** [comment]
Need `wire cutters`! Yep!
And something for [multi rearranging wires](https://github.com/Regalis11/Barotrauma/discussions/7327).

**Reply by Krevetka7350** [reply]
I like the concept of being able to edit wiring without dragging a wire across the ship once again but i often use multiple components which just rest on top of some wire. There definitely should be an option to "cut" or not to.

**Comment by MaaiG** [comment]
If we need `wire cutters`, we need a `soldering iron`...
